mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -227,6 +243,11 @@ def ping():  # pylint: disable=unused-variable
         status = 200 if health else 404
         return flask.Response(response="\n", status=status, mimetype="application/json")
 
+    @app.route("/version/<module>", methods=["GET"])
+    def get_module_version(module):
+        ver = importlib.import_module(module).__version__
+        return flask.Response(response=ver, status=200, mimetype="text/plain")
```

Comment:
Another way to get a module version is to create a custom model whose predict method returns the version of a specified module.

Reply:
> create a custom model

in order to load custom model in the "created python env", we need load the module which defines the custom model class. For databricks/universe integration test, this is hard. So this helper method helps tests in databricks/universe side.

Reply:
@WeichenXu123 Can we remove this `get_module_version()` method?

Reply:
@WeichenXu123 This method is still here. Can we remove it?

Reply:
removed!